Remotion/Web/Core/UI/SmartPage.cs

```diff
@@ -116,7 +116,7 @@ string ISmartPage.CheckFormStateFunction
   [Description("The message displayed when the user attempts to leave the page.")]
   [Category ("Appearance")]
   [DefaultValue ("")]
-  public virtual string AbortMessage 
+  public virtual string? AbortMessage 
   {
     get { return _smartPageInfo.AbortMessage; }
```

Comment:
This kind of API is just bad design. Very sad. very very sad. Want to go back to 2006 and use NRT right then and there...